src/jaxsim/api/contact.py

```diff
@@ -167,11 +167,11 @@ def estimate_model_height(model: Model.JaxSimModel) -> jtp.Float:
             model=model, soft_contacts_params=soft_contacts.SoftContactsParams()
         )
 
-        W_p_CoM = Model.com_position(model=model, data=zero_data)
+        W_p_CoM = Model.com_position(model=model, data=zero_data)[2]
 
         if model.physics_model.is_floating_base:
             W_pz_C = collidable_point_positions(model=model, data=zero_data)[:, -1]
-            return 2 * (W_p_CoM[2] - W_pz_C.min())
+            return 2 * (W_p_CoM - W_pz_C.min())
```

Comment:
```suggestion
            return 2 * (W_pz_CoM - W_pz_C.min())
```

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/98/commits/d378d565d77d6d156ed16bbc8499444478d3fa95